enterprise/admin/views.py

```diff
@@ -1021,9 +1021,6 @@ def delete(self, request, customer_uuid):
             message = _("Email {email} is not associated with Enterprise "
                         "Customer {ec_name}").format(
                             email=email_to_unlink, ec_name=enterprise_customer.name)
-            return HttpResponse(message, content_type="application/json", status=404)
+            return HttpResponse(message, status=404)
```

Comment:
I think this change came from an overzealous pylint-django check; this changes the behavior from returning a 404 with JSON content to a 404 with HTML content, which may be a problem.  Should probably leave this as it was with a comment to disable the check that was failing.